build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -13,14 +13,14 @@ namespace Microsoft.DotNet.Cli.Build
     public class DebTargets
     {
         [Target(nameof(GenerateSdkDeb))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
Really?  Why don't we want to build .deb packages for 16.04?

I would think if we have specific assemblies for Ubuntu 16, we would need Ubuntu 16 specific deb packages.

@brthor